README.md

```diff
@@ -432,6 +433,7 @@ window.esmsInitOptions = {
   fetch: (url, options) => fetch(url, options), // default is native
   revokeBlobURLs: true, // default false
   enforceIntegrity: true, // default false
+  override: true, // default false
```

Comment:
Let's give it a more descriptive name, perhaps something like `mapOverrides: true`?

Reply:
Yep, sounds good, went a bit overboard with the bytes saving here. 😅

Reply:
Addressed here: https://github.com/guybedford/es-module-shims/pull/257/commits/39432375f5d59f61271f1d5bf8b0360322c4a54d